qtranslate.php

```diff
@@ -54,7 +54,7 @@
 	ja by Brian Parker
 	nl by RobV
 	pt by netolazaro
-	pt-br by Pedro Mendonça
+	pt-pt by Pedro Mendonça
```

Comment:
There are two syntaxes as I've explained above:
According Locale Codes the language is defined by 'pt' (Portugal) and 'pt-br' (Brazil)
According WordPress Locale the language is defined by 'pt_PT' (Portugal) and 'pt_BR' (Brazil)

All my contributions are to native Portugal (pt or pt_PT according the above locale codifications).

There was already a single 'pt' in 'qtranslate.php' that was incorrecly set as generic 'pt' but with brazilian flag etc
I corrected it to be well refferenced as pt-br and added to the list the actual native 'pt(_PT)' from Portugal that I'm contributing.
I think I cannot explain better then on my previous comment :)